tests/acceptance/bootstrap/ArchiverContext.php

```diff
@@ -325,4 +325,27 @@ public function theDownloadedArchiveShouldContainTheseFiles(string $type, TableN
 		\unlink($tempFile);
 		$this->removeDir($tempExtractFolder);
 	}
+
+	/**
+	 * @Then user :user should be able to download archive of federated shared folder :resource
+	 *
+	 * @param string $user
+	 * @param string $resource
+	 *
+	 * @return void
+	 */
+	public function userShouldBeAbleToDownloadArchiveFederatedSharedFolder(string $user, string $resource): void {
```

Comment:
Just matching the step-def
```suggestion
	public function userShouldBeAbleToDownloadArchiveOfFederatedSharedFolder(string $user, string $resource): void {
```